implementations/implementation.hbs

```diff
@@ -0,0 +1,56 @@
+    <section id='abstract'>
+      <p>
+This is the most recent implementation report for the
+<a href="https://tools.ietf.org/html/draft-cavage-http-signatures">HTTP Signatures</a> specification.
+      </p>
+    </section>
+
+    <section id='sotd'>
+      <p>
+Comments regarding this document are welcome. Please file issues
+directly on <a href="https://github.com/w3c-dvcg/http-signatures-test-suite">GitHub</a>,
+or send them to
+<a href="mailto:public-credentials@w3.org">public-vc-comments@w3.org</a>
+(<a href="mailto:public-credentials@w3.org?subject=subscribe">subscribe</a>,
+<a href="https://lists.w3.org/Archives/Public/public-credentials/">archives</a>).
+      </p>
+
+    </section>
+
+    <section class="informative">
+      <h1>Introduction</h1>
+
+      <p>
+The purpose of this document is to demonstrate that there are multiple
+interoperable implementations of HTTP Signatures that are capable of generating
+output that is conformant to the latest version.
+      </p>
+
+      <section class="informative">
+        <h2>Testing Methodology</h2>
+
+        <p>
+The testing framework for HTTP Signatures executes the
+following process for every conformance statement in the 
+<a href="https://tools.ietf.org/html/draft-cavage-http-signatures">
+Http Signatures specification</a> :
```

Comment:
Http -> HTTP

Reply:
Addressed in https://github.com/w3c-dvcg/http-signatures-test-suite/pull/5/commits/f791a4746422086bc8a91bda4d1292eb935eea51